lte/protos/session_manager.proto

```diff
@@ -28,6 +28,11 @@ message RuleRecord {
   uint64 bytes_rx = 4;
   string ue_ipv4 = 5;
   string ue_ipv6 = 6;
+  enum PolicyType {
+    REGULAR_POLICY = 0;
+    DEFAULT_DROP_POLICY = 1;
+  }
+  PolicyType type = 7;
```

Comment:
@uri200 how dose this look for you in terms of sessiond processing down the line

Reply:
Also the rule name will be 'deafult_drop_flow'

Reply:
per what we discussed my suggestion is to use `bytes_tx_dropped` and `bytes_rx_dropped` to make the message more generic. 

Or alternatively, you can do the same you did, but instead of using the particular case of an "" rule, just use a field to indicate that record is for dropped packets. This way you can include `DENY` rules on the messages. But my concern with that is we are using `bytes_tx` and `bytes_rx` for allowed packets and denied packets.
```
 enum RecordType {
    DROPPED = 0;
    ALLOWED = 1;
  }
```

Reply:
Also, sessiond doesnt terminate the session until pipelined stops reporting records for that session. IE, when a session has to be terminated, it is not terminated until we do NOT receive any record for that session. If now, pipelined is sending records for dropped packages that will prevent termination. 

We may need to add some logic in SessionD once this format is decided